InfiniteLoopLMSSQL/src/com/st/lmssql/models/Borrower.java

```diff
@@ -0,0 +1,54 @@
+package com.st.lmssql.models;
+
+public class Borrower {
+	private int cardNo;
+	private String name;
+	private String address;
+	private String phone;
+	
+	public Borrower() {}
+	
+	public Borrower(int cardNo, String name, String address, String phone) {
+		this.cardNo = cardNo;
+		this.name = name;
+		this.address = address;
+		this.phone = phone;
+	}
+
+	public int getCardNo() {
+		return cardNo;
+	}
+
+	public void setCardNo(int cardNo) {

```

Comment:
You may want to consider making the id final, so once the object is made, no one can change it.